### PR TITLE
Removed redundant parent Linear layout in drawer_list_item.xml

### PR DESCRIPTION
**Pull request by b-ade**

### CODE REVIEW

File: omniNotes/src/main/res/layout/drawer_list_item.xml [hunk 1]
```diff
@@ -15,13 +15,7 @@
...
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:gravity="center_vertical"
-  android:orientation="vertical">
-
-  <LinearLayout
+    xmlns:pixlui="http://schemas.android.com/apk/com.neopixl.pixlui"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:gravity="center_vertical"
```
Comment (b-ade) [inline omniNotes/src/main/res/layout/drawer_list_item.xml:21]: The old outter linear layout seems pretty similar to this

Comment (b-ade) [inline omniNotes/src/main/res/layout/drawer_list_item.xml:21]: Seems pretty similar to the old outer linear layout

**Review by b-ade — COMMENTED** [review]
o

**Comment by federicoiosue** [comment]
Sorry for missing your PR, I've included your modification [here](https://github.com/federicoiosue/Omni-Notes/commit/bef0e8ebae4c892abdeb2d11badd1e458008046f)

**Pull request closed without merging**